src/plugins/chart_expressions/expression_heatmap/public/components/heatmap_component.test.tsx

```diff
@@ -27,6 +27,17 @@ jest.mock('@elastic/charts', () => {
   };
 });
 
+const actWithTimeout = (action: Function, timer: number = 1) =>
```

Comment:
we implemented similar solutions in couple places already, maybe it's worth to unify. For example, in lens in `lens/public/editor_frame_service/editor_frame/config_panel/config_panel.test.tsx` we do it this way:
```
// helper function
const waitMs = (time: number) => new Promise((r) => setTimeout(r, time));

// usage from the first test:
 await waitMs(0);
 await act(async () => {
    await component.update();
 });

Reply:
🤷 , probably yes, but it's just super simple logic, maybe no sense to unify it.  All the same, tests should have some part of independence.